test/comparator/test_field_comparator.py

```diff
@@ -321,6 +321,37 @@ def test_resource_reference_addition_non_breaking(self):
         self.assertEqual(finding.category.name, "RESOURCE_REFERENCE_ADDITION")
         self.assertEqual(finding.change_type.name, "MINOR")
 
+    def test_resource_reference_addition_non_breaking2(self):
```

Comment:
Maybe change the test name to indicate this tests child type resource ref additions.
```suggestion
    def test_resource_reference_child_type_addition_non_breaking(self):
```

Reply:
Thanks! I have concerns to make the test name too long, but it makes sense to indicate the intentions for a specific unit test.

Reply:
Done.